.github/workflows/isort.yml

```diff
@@ -1,10 +1,10 @@
 name: Code Formatter
 
+# Add `pull_request:` to apply on active PRs 
```

Comment:
Could you please change this to:

`# Add 'pull_request:' beneath 'on:' to apply on active PRs`